packages/transforms/snapshot/src/index.ts

```diff
@@ -0,0 +1,42 @@
+import { GraphQLSchema } from "graphql";
+import { TransformFn, YamlConfig } from '@graphql-mesh/types';
+import { extractResolversFromSchema, composeResolvers, ResolversComposerMapping, ResolversComposition } from '@graphql-toolkit/common';
+import { addResolveFunctionsToSchema } from 'graphql-tools-fork';
+import { isAbsolute, join } from 'path';
+import { existsSync, readFileSync, writeFileSync } from "fs";
+import { ensureFileSync } from 'fs-extra';
+import md5 from 'md5';
+
+const snapshotTransform: TransformFn<YamlConfig.SnapshotTransformConfig> = async ({
+    schema,
+    config
+  }): Promise<GraphQLSchema> => {
+    const resolvers = extractResolversFromSchema(schema);
```

Comment:
Maybe add a check and throw an error in case the environment is doesn't support fs?